### PR TITLE
fix some outdated entries in glossary

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

fixes some out of date entries in the glossary

### CODE REVIEW

File: src/content/glossary/index.md [hunk 1]
```diff
@@ -794,11 +791,11 @@ An attack that consists of an attacker contract calling a victim contract functi
 
 ### reward {#reward}
 
-An amount of ether included in each new block as a reward by the network to the [miner](#miner) who found the [proof-of-work](#pow) solution.
+An amount of ether [awarded to validators](developers/docs/consensus-mechanisms/pos/rewards-and-penalties) that perform certain functions, including proposing a block or participating in a sync-committee, in each slot.
```
Comment (wackerow) [inline src/content/glossary/index.md:794]:
```suggestion
An amount of ether [awarded to validators](/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/) that perform certain functions, including proposing a block or participating in a sync-committee, in each slot.
```

File: src/content/glossary/index.md [hunk 2]
```diff
@@ -679,7 +680,7 @@ The collective [hashrate](#hashrate) produced by an entire mining network. Minin
 
 ### non-fungible token (NFT) {#nft}
 
-Also known as a "deed," this is a token standard introduced by the ERC-721 proposal. NFTs can be tracked and traded, but each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
+An instance of one of several contract standards, commonly [ERC-721](developers/docs/standards/tokens/erc-721) or [ERC-1155](developers/docs/standards/tokens/erc-1155). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
```
Comment (wackerow) [inline src/content/glossary/index.md:683]:
```suggestion
An instance of one of several contract standards, commonly [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
```

File: src/content/glossary/index.md [hunk 3]
```diff
@@ -794,11 +791,11 @@ An attack that consists of an attacker contract calling a victim contract functi
 
 ### reward {#reward}
 
-An amount of ether included in each new block as a reward by the network to the [miner](#miner) who found the [proof-of-work](#pow) solution.
+An amount of ether [awarded to validators](developers/docs/consensus-mechanisms/pos/rewards-and-penalties) that perform certain functions, including proposing a block or participating in a sync-committee, in each slot.
 
 ### Recursive Length Prefix (RLP) {#rlp}
 
-An encoding standard designed by the Ethereum developers to encode and serialize objects (data structures) of arbitrary complexity and length.
+An [encoding standard](developers/docs/data-structures-and-encoding) designed by the Ethereum developers to encode and serialize objects (data structures) of arbitrary complexity and length.
```
Comment (wackerow) [inline src/content/glossary/index.md:798]:
```suggestion
An [encoding standard](/developers/docs/data-structures-and-encoding/) designed by the Ethereum developers to encode and serialize objects (data structures) of arbitrary complexity and length.
```

**Comment by konopkja** [comment]
@minimalsm @jmcook1186 @wackerow i think we need to reconsider the purpose of these glossary terms, especially the basic terms such as "account", "address"  and probably a few others are too important to be described in a tech spec style like it is now. 

Especially if we are to use the glossary mouse over widget more, these explanations will be more visible and must be up to the task broadly relevant and useful to a non tech person.

File: src/content/glossary/index.md [hunk 4]
```diff
@@ -173,19 +172,19 @@ Converting code written in a high-level programming language (e.g., [Solidity](#
...
+A group of at least 128 [validators](#validator) assigned to validate blocks in each slot. One of the validators in the committee is the aggregator, responsible for aggregating the signatures of all other validators in the committee that agree on an attestation. Committees rotate such that every active validator attests in every [epoch](#epoch). Not to be confused with [sync committee](#sync-committee).
 
 ### computational infeasibility {#computational-infeasibility}
 
 A process is computationally infeasible if it would take an impracticably long time (e.g. billions of years) to do it for anyone who might conceivably have an interest in carrying it out.
 
 ### consensus {#consensus}
 
-When a supermajority of nodes on the network all have the same blocks in their locally validated best blockchain. Not to be confused with [consensus rules](#consensus-rules).
+When a supermajority of nodes on the network all have the same blocks in their local copy of the blockchain. Not to be confused with [consensus rules](#consensus-rules).
```
Comment (konopkja) [inline src/content/glossary/index.md:183]: what is a supermajority? can we explain better?

File: src/content/glossary/index.md [hunk 5]
```diff
@@ -587,7 +588,7 @@ Cryptographic [hash](#hash) function used in Ethereum. Keccak-256 was standardiz
 
 ### layer 2 {#layer-2}
 
-An area of development focused on layering improvements on top of the Ethereum protocol. These improvements are related to [transaction](#transaction) speeds, cheaper [transaction fees](#transaction-fee), and transaction privacy.
+An area of development focused on moving transactions off of Ethereum layer-1 to enable Ethereum to scale. These improvements reduce transaction fees for users.
```
Comment (konopkja) [inline src/content/glossary/index.md:591]: switch layer 1 lingo to mainnet?

File: src/content/glossary/index.md (hunk 2 above)
Comment (konopkja) [reply src/content/glossary/index.md:683]: "An instance of one of several contract standards"  is super difficult to comprehend, we need better explanation

**Comment by corwintines** [comment]
@konopkja This makes me think of a dual-purpose glossary. Have a technical and an ELI5 glossary maybe? Future idea, your comment just sparked that idea.

File: src/content/glossary/index.md [hunk 6]
```diff
@@ -27,12 +27,11 @@ An object containing an [address](#address), balance, [nonce](#nonce), and optio
 
 ### address {#address}
 
-Most generally, this represents an [EOA](#eoa) or [contract](#contract-account) that can receive (destination address) or send (source address) [transactions](#transaction) on the blockchain. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).
+A string of 64 characters representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/content/glossary/index.md:30]: Agree with 40 change here, I think this is related to private key, but address should be 40. 

@konopkja agree that we could have a simpler definition here as well. Going to commit @wackerow changes, but can propose something here after.

File: src/content/glossary/index.md [hunk 7]
```diff
@@ -27,12 +27,11 @@ An object containing an [address](#address), balance, [nonce](#nonce), and optio
 
 ### address {#address}
 
-Most generally, this represents an [EOA](#eoa) or [contract](#contract-account) that can receive (destination address) or send (source address) [transactions](#transaction) on the blockchain. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).
+A string of 40 characters representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).
```
Comment (corwintines) [inline src/content/glossary/index.md:30]:
```suggestion
A string of characters starting with `0x` representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. 
```

I think this could be a more simplified version. Let me know what you think @konopkja

File: src/content/glossary/index.md (hunk 4 above)
Comment (corwintines) [reply src/content/glossary/index.md:183]:
```suggestion
When a supermajority (more than 66%) of nodes on the network all have the same blocks in their local copy of the blockchain. Not to be confused with [consensus rules](#consensus-rules).
```

@konopkja adding a change that should address this, let me know what you think

File: src/content/glossary/index.md (hunk 5 above)
Comment (corwintines) [reply src/content/glossary/index.md:591]:
```suggestion
An area of development focused on moving transactions off of Ethereum Mainnet to enable Ethereum to scale. These improvements reduce transaction fees for users.
```

File: src/content/glossary/index.md [hunk 8]
```diff
@@ -679,7 +680,7 @@ The collective [hashrate](#hashrate) produced by an entire mining network. Minin
 
 ### non-fungible token (NFT) {#nft}
 
-Also known as a "deed," this is a token standard introduced by the ERC-721 proposal. NFTs can be tracked and traded, but each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
+An instance of one of several contract standards, commonly [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
```
Comment (corwintines) [inline src/content/glossary/index.md:683]:
```suggestion
A non-fungible token (NFT) is a one-of-a-kind digital asset that represents ownership of a specific item or piece of content. Common contracts for NFT's include [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/).
```

@konopkja tried to improve this like you suggested

**Review by corwintines — COMMENTED** [review]
Merged some of the suggestions, and provided a few where asked for. 

@konopkja @wackerow @jmcook1186 if we can agree on these, ill merge dev in and fix this PR to use the new glossary structure.

File: src/content/glossary/index.md (hunk 2 above)
Comment (konopkja) [reply src/content/glossary/index.md]: @corwintines i think the order in your explanation is wrong. It should start with explanation of uniqueness and end with saying there are few versions of these standards.

Comment (konopkja) [reply src/content/glossary/index.md]: NFTs can represent ownership of digital or physical assets. Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). There are two smart contract standards used for NFTs, [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/).

File: src/content/glossary/index.md (hunk 7 above)
Comment (corwintines) [reply src/content/glossary/index.md:30]:
```suggestion
A unique identifier that represents your account. All accounts on Ethereum have a public address that starts with `0x`. This address is sharable and used to receive assets to your account. 
```

@konopkja put together in call

**Comment by konopkja** [comment]
@minimalsm went through the list of changes and compiled this list that should be added:

ABI
block validation
bootnode
bytecode
committee
consensus client
finality
fork
fork-choice algorithm
fraud proof
Merkle Patricia trie
ommer (uncle) block
proof-of-work
reward
Recursive Length Prefix (RLP)
shard / shard chain
terminal total difficulty (TTD)

**Comment by minimalsm** [comment]
Thanks @konopkja, can you create an issue to implement these in our new JSON structure?

Closing this PR out as the structure of the glossary has fundamentally changed.
